Makefile

```diff
@@ -15,7 +15,7 @@ strip: build
 
 .PHONY: build-podman
 build-podman:
-	podman build .
+	podman build -t provisioning-backend -f build/Dockerfile .
```

Comment:
I use podman remote build (podman connects to a remote box where podman is daemonized and performs the actual build) and this fails for some reason (perhaps a bug in the protocol).

Could you create a symlink so I can still build it manually by issusing `podman build .`? Do this:

    ln -f build/Dockerfile Containerfile

And commit the symlink into git, if there are no objections.